recovery.go

```diff
@@ -86,6 +87,11 @@ func (h recoveryHandler) log(v ...interface{}) {
 	}
 
 	if h.printStack {
-		debug.PrintStack()
+		stack := debug.Stack()
+		if h.logger != nil {
+			h.logger.Println(string(stack))
+		} else {
+			_, _ = os.Stderr.Write(stack)
```

Comment:
I would expect this to print to log as is done above. Is there a reason this prints directly to stderr instead?

Reply:
oh there is no reason as such. Good point, let me update it.